genesyscloud/resource_genesyscloud_architect_schedules.go

```diff
@@ -93,9 +96,10 @@ func ResourceArchitectSchedules() *schema.Resource {
 				ValidateDiagFunc: validateLocalDateTimes,
 			},
 			"rrule": {
-				Description: "An iCal Recurrence Rule (RRULE) string.",
-				Type:        schema.TypeString,
-				Optional:    true,
+				Description:      "An iCal Recurrence Rule (RRULE) string.",
+				Type:             schema.TypeString,
```

Comment:
I noticed there is an extra bit of info in the description on API explorer: "An iCal Recurrence Rule (RRULE) string. It is required to be set for schedules determining when upgrades to the Edge software can be applied." Probably no harm to add it in